docs/python.org/code-of-conduct/Transparency-Report-Guidelines.md

```diff
@@ -0,0 +1,45 @@
+# Releasing a Good Conference Transparency Report
+
+Publishing a transparency report after the event concludes is a critical step for building trust within the community and ensuring the community that any issues that have been reported have been handled in an appropriate manner.
+
+This guide is provided for conference organizers. We particularly hope it is useful for Conference Chairs and Code of Conduct responders.
+
+In this document we're discussing the **public-facing** summary transparency report. For operational purposes, an event should also keep private records of all reports and issues at the event.
+
+
+## When to release
+
+A timely report helps everybody:
+
+- Reporters know that their reports were taken seriously;
+- Attendees can see that CoC reports are an integral part and so feel confident to attend in the future and report issues they themselves witness;
+- Organizers demonstrate strong commitment to follow their stated goals and procedures.
+
+Since it's important to be able to recall details of covered issues accurately, a timely report will be of higher quality.
+
+Releasing an initial transparency report should be part of the closing ceremony. Following up with affected parties might take time after the event. Therefore, a separate complete report can be provided in writing at a later time. It wouldn't take more than a month after the event to close the loop on the final report.
```

Comment:
My brain is tired, but my gut feeling is we should lean harder on recommending and suggesting using "should" and "would."

> Releasing an initial transparency report should be part of the closing ceremony.

Something like, "We recommend including an initial transparency report as part of the closing ceremony." 

> It wouldn't take more than a month after the event to close the loop on the final report.

I struggled with this sentence. It feels like something we should reword, "We recommend closing the loop on final reports within one month after the event when possible." If we are being 100% transparent here, we have seen reports from conferences take as much as a year or more to fully resolve.